front/src/applications/operationalStudies/views/Project.tsx

```diff
@@ -118,7 +113,7 @@ export default function Project() {
         };
         try {
           const filteredData = await postSearch(payload).unwrap();
-          let filteredStudies = [...filteredData] as SearchStudyResult[];
```

Comment:
Note: this conflicts with #5464

Reply:
I’ll wait for the merge to handle the conflict